boudicca.base/eventdb-openapi/src/main/resources/api-docs.json

```diff
@@ -1 +1,349 @@
\ No newline at end of file
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Boudicca EventDB",
+    "license": {
+      "name": "GPL v3"
+    },
+    "version": "0.1"
```

Comment:
where is the version defined?
It should be somewhere in the gradle, but i couldn't find it.

Reply:
found it. still it's different from the gradle defined api version

Reply:
this is from your changes, i have not touched this file and the generation itself. there are still things to do better, especially because we do not want to check in the .json file. and if possible it would be nice to publish it as well. but this is for later, not this MR